build/configs/imxrt1020-evk/imxrt1020-evk_download.sh

```diff
@@ -246,6 +247,13 @@ ERASE)
 		shift
 	done
 	;;
+#Download SMARTFS Partition
+userfs)
```

Comment:
Isn't it working by `*)` case?
@btheosam Could you confirm?

Reply:
This block of changes are NOT needed as userfs is covered in *) case. 
 *) case covers all partitions present in the config file.

Reply:
Yes, i agree, have to change this.